tmt/plugins/__init__.py

```diff
@@ -146,13 +148,26 @@ def _explore_entry_points(logger: Logger) -> None:
     _explore_entry_point(ENTRY_POINT_NAME, logger.descend())
 
 
-def explore(logger: Logger) -> None:
-    """ Explore all available plugin locations """
+def explore(logger: Logger, again: Optional[bool] = False) -> None:
```

Comment:
`Optional` seems to be unnecessary - when `again` is left out, it'd be set to `False`, therefore it would even then be a boolean. `again: bool = False` would be a perfectly valid signature, and even better as it's stricter that the `Optional` one, removing the space of `again=None`.

Reply:
Ah, this got me again! Thanks for clarification (again) :)

Reply:
Fixed in 57febb9dc38f7663cc32af4ad58f087d0e27d77d.